draft-js-inline-toolbar-plugin/src/index.js

```diff
@@ -35,9 +35,10 @@ export default (config = {}) => {
   };
 
   return {
-    initialize: ({ getEditorState, setEditorState }) => {
+    initialize: ({ getEditorState, setEditorState, getEditorRef }) => {
       store.updateItem('getEditorState', getEditorState);
       store.updateItem('setEditorState', setEditorState);
+      store.updateItem('getEditorRef', getEditorRef);
```

Comment:
ugh it makes me sad that we have our own abstraction for state management in plugins...